src/style.css

```diff
@@ -1 +1,148 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use `*` selector (it impacts performance). Set styles only for elements that require them.